test/Superpower.Tests/Parsers/SpanTests.cs

```diff
@@ -83,5 +84,33 @@ public void RegexMatches()
             var r = parser(input);
             Assert.Equal("Foo", r.Value.ToStringValue());
         }
+
+        [Theory]
+        [InlineData("", "")]
+        [InlineData("STOP", "")]
+        [InlineData("123STOP", "123")]
+        [InlineData("STOP123", "")]
+        [InlineData("123STOP123STOP123", "123")]
+        public void UntilMatches(string text, string expected)
```

Comment:
Could rename to be more descriptive - `UntilMatchesWhenStopwordIsPresent()` perhaps? (Similar for the tests below?)

Reply:
Done